package.json

```diff
@@ -45,7 +45,7 @@
     "eslint-config-keep": "github:keep-network/eslint-config-keep#0c27ade",
     "ethereum-waffle": "^3.3.0",
     "ethers": "^5.0.32",
-    "hardhat": "^2.1.1",
+    "hardhat": "^2.2.1",
```

Comment:
If `yarn.lock` resolves to `2.6.1`, why don't we apply this version here as well?

Reply:
Oh, I meant `^2.6.1`

Reply:
Done in https://github.com/keep-network/coverage-pools/pull/178/commits/841db95b1bf9cac977338b757d9f576b8d26fe2d.